### DISCUSSION TITLE
How to change underline color for active tab of the TabControl?

### DISCUSSION THREAD

**Discussion by TangDave** [AvaloniaUI/Avalonia#16967] [Q&A] [answered]
I see that the default underline color of the active tab is something like teal - is there a property I can set to change this?

**Comment by TangDave** [comment, marked as answer]
Found the answer in this thread: https://github.com/AvaloniaUI/Avalonia/discussions/13401

```
		<Style Selector="TabItem:selected /template/ Border#PART_SelectedPipe">
			<Setter Property="Background" Value="MidnightBlue"/>
		</Style>
```

**Reply by Suchacoolguy** [reply]
Helped a lot. Thank you for sharing :)
